_gsocorgs/2018/uoslo.md

```diff
@@ -1,8 +1,11 @@
 ---
 title: "University of Oslo"
-author: "Mario Lassnig"
+author: "Cédric Serfon"
 layout: default
-organization: UOslo
+organization: UiO
+logo: University-of-Oslo.png
```

Comment:
Hi Cedric, thanks for the changes.
I noticed there is already a logo image of UOslo in the repo:
https://github.com/HEP-SF/hep-sf.github.io/blob/master/images/UOslo.png

If that image is ok for you, I would propose to remove the logo from the PR and refer to UOslo.png from your uoslo.md file.

Reply:
I think the one you mention is an old one.